### DISCUSSION TITLE
Can I use alerts in my posts?

### DISCUSSION THREAD

**Discussion by ErikKalkoken** [adityatelange/hugo-PaperMod#1587] [Q&A] [answered]
I would like to use alerts/callouts in my posts. Github recently introduced those as part of their markdown syntax.

Here is an example:

> [!TIP]
> Optional information to help a user be more successful.

Is this possible with the PaperMod theme / hugo?

See also: [An option to highlight a "Note" and "Warning" using blockquote (Beta)](https://github.com/orgs/community/discussions/16925)

**Comment by russmckendrick** [comment, marked as answer]
I use this additional theme alongside PaperMod on my blog: https://github.com/martignoni/hugo-notice/. You can see it in action in the following posts.

- https://www.russ.cloud/2024/06/16/day-to-day-tools-the-2024-edition/
- https://www.russ.cloud/2024/04/02/updating-my-dotfiles/

**Reply by ErikKalkoken** [reply]
Thank you, that looks great. I will try it out.
